orc/src/main/java/org/apache/iceberg/orc/ORC.java

```diff
@@ -145,6 +146,7 @@ public WriteBuilder metricsConfig(MetricsConfig newMetricsConfig) {
 
     public <D> FileAppender<D> build() {
       Preconditions.checkNotNull(schema, "Schema is required");
+      parseWriterCustomizedConf(conf);
```

Comment:
it seems not necessary to have a separated method for this

Reply:
removed

Reply:
@jackye1995  Would it be more reasonable to add a common method to resolve all configuration items, converting the specific name in Iceberg to the native name of orc? In this way, users can use all orc configuration items, not only those listed in properties.
The method may like this:

```
  private void setAllOrcConf(Configuration conf) {
    Arrays.stream(OrcConf.values()).forEach(oneConf -> {
      String attribute = oneConf.getAttribute();
      String icebergPropertiesName =
          ("write." + attribute).replace(".", "-")
              .replaceFirst("-", ".")
              .replaceFirst("-", ".");
      if (conf.get(icebergPropertiesName) != null) {
        conf.set(attribute, conf.get(icebergPropertiesName);
      }
    });
  }
```